packages/app-next/app-config.yaml

```
@@ -16,6 +16,9 @@ app:
         config:
           filter: kind:component has:links
     - entity-card:linguist/languages
+    - entity-card:catalog-graph/entity-relations:
```

```suggestion
    - entity-card:catalog-graph/relations:
```

Could this make sense? It reads well and the `entity-` part might be a bit redundant